sp/src/game/server/ez2/npc_arbeit_personality_core.h

```diff
@@ -0,0 +1,167 @@
+//=============================================================================//
+//
+// Purpose:		Portal 2 Personality Core NPC created from scratch in Source 2013
+//				to fit into the world and mechanics of Half-Life 2. It has the same
+//				response and choreography abilities as other ally NPCs and is based
+//				largely on the Wilson NPC.
+// 
+//				This was originally created for Entropy : Zero 2.
```

Comment:
Ditto the comment from the crab synth PR

Maybe we could phrase it as "This was originally created for possible use in Entropy : Zero 2 post-release content." and use that phrasing for the assassins, husks, personality cores, crab synth, etc? I know it's a bit of a mouthful but I think there's a lot of confusion about that right now.